pkg/utils/type_cast.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"reflect"
+
+	"github.com/juju/errors"
+)
+
+func CastToSlice(arg interface{}) (out []interface{}, ok bool) {
```

Comment:
Why not cast directly? out, ok := arg.([]interface{})

Reply:
just to make sure that `arg` can be any type of slice, for example `[]string`, `[]interface{}`, useful for both test and actual code.